src/fparser/two/tests/fortran2008/test_parent_identifier_r1118.py

```diff
@@ -41,81 +41,82 @@
    descendant of that module.
 
    It is not possible to test C1113 with fparser as the names may be
-   in different files
+   in different files.
 
 '''
 import pytest
 from fparser.two.Fortran2003 import NoMatchError
 from fparser.two.Fortran2008 import Parent_Identifier
 from fparser.two.parser import ParserFactory
-# this is required to setup the fortran2008 classes
-_ = ParserFactory().create(std="f2008")
 
 
 def test_simple_1():
-    '''Test the parsing of a minimal parent-identifier statement'''
+    '''Test the parsing of a minimal parent-identifier statement.'''
+    _ = ParserFactory().create(std="f2008")
```

Comment:
Again, please could you do this just once in a fixture in this file.

Reply:
Done.